src/index.js

```diff
@@ -57,8 +58,20 @@ async function main() {
       core.setFailed(`Assertions have failed.`)
       // continue
     }
+
+    const slackWebhookUrl = input.slackWebhookUrl
```

Comment:
Thought: I almost consider this "outputting" to slack as "uploading". I think it might belong in the uploading section?

Reply:
Fairly saying I dunno. Because Slack is like showing errors in console but on steroids :) 
In other hand it also make sense to put it to uploading section :)